server/routes/adjustmentRoutes.ts

```diff
@@ -331,6 +331,57 @@ export default class AdjustmentRoutes {
     })
   }
 
+  public unusedDeductionDays: RequestHandler = async (req, res): Promise<void> => {
+    const { nomsId, addOrEdit } = req.params
+    const { prisonerNumber } = res.locals.prisoner
+    const { username } = res.locals.user
+    const { bookingId } = res.locals.prisoner
+
+    const startOfSentenceEnvelope = await this.prisonerService.getStartOfSentenceEnvelope(bookingId, username)
+    const adjustments = await this.adjustmentsService.findByPerson(
+      nomsId,
+      startOfSentenceEnvelope.earliestSentence,
+      username,
+    )
+
+    const days =
+      addOrEdit === 'edit'
+        ? adjustments.filter(it => it.source !== 'NOMIS').find(it => it.adjustmentType === 'UNUSED_DEDUCTIONS')
+            ?.effectiveDays || 0
+        : 0
+
+    const totalRemandAndTaggedBailDays =
+      adjustments
+        .filter(it => it.adjustmentType === 'TAGGED_BAIL' || it.adjustmentType === 'REMAND')
+        ?.map(it => it.effectiveDays)
+        .reduce((acc, cur) => {
+          return acc + cur
+        }, 0) || 0
+
+    return res.render('pages/adjustments/unused-deductions/days', {
+      model: new UnusedDeductionsDaysModel(
+        prisonerNumber,
+        addOrEdit,
+        UnusedDeductionsDaysForm.fromDays(days, totalRemandAndTaggedBailDays, addOrEdit),
+      ),
+    })
+  }
+
+  public submitUnusedDeductionDays: RequestHandler = async (req, res): Promise<void> => {
+    const { nomsId, addOrEdit } = req.params
+    const { prisonerNumber } = res.locals.prisoner
+    const unusedDeductionsDaysForm = new UnusedDeductionsDaysForm({ ...req.body })
+    await unusedDeductionsDaysForm.validate()
+    if (unusedDeductionsDaysForm.errors.length) {
+      return res.render('pages/adjustments/unused-deductions/days', {
+        model: new UnusedDeductionsDaysModel(prisonerNumber, addOrEdit, unusedDeductionsDaysForm),
+      })
+    }
+
+    // TODO: Submit to backend
```

Comment:
When is this TODO being done? I think this is the final ticket

Reply:
Yep, that's the final ticket. The confirm and save page. It will be done as soon as this is merged.

What's our stance on TODOs?